validator/src/main/resources/expected-data-template/java/eks/rds-mysql-trace.mustache

```diff
@@ -73,8 +79,7 @@
                                 "K8s.Namespace": "^{{appNamespace}}$",
                                 "PlatformType": "^AWS::EKS$",
                                 "aws.span.kind": "^CLIENT$",
-                                "db.sql.table": "^tables$",
-                                "db.operation": "^SELECT$"
```

Comment:
May I know why do we delete this?

Reply:
I actually removed that as part of previous template, I can add it back.

Reply:
Updated the template.